BC-NOV-20-Proyectos/alerta-covid-web-team02
# Iss22 Modeling

# Description
 - Welcome page created with a responsive design for mobile devices
 
 # Testing
- Open your terminal and run `rails s` to set the rails server up.
- Go to [](http://localhost:3000/ ) to see the welcome page
 
  ## Screenshots 
![Screenshot from 2021-04-15 15-40-35](https://user-images.githubusercontent.com/47892542/114935744-2d791d00-9e01-11eb-9bf5-127a9b90d1f0.png)
![Screenshot from 2021-04-15 15-41-31](https://user-images.githubusercontent.com/47892542/114935748-2fdb7700-9e01-11eb-9894-a2fa241d5399.png)
